setup.cfg

```diff
@@ -24,19 +24,18 @@ classifiers =
 [options]
 include_package_data = True
 packages = find:
-python_requires = >=3.7
+python_requires = >=3.9
```

Comment:
Unless I am mistaken, for the moment, we don't increase this. There are many non-InvenioRDM installation using various Invenio module out there and unless a specific Python version is really needed, we tend to avoid increasing it.

Reply:
Ok, didn't think about that, I'll change back.